etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/orders/SendOrderUseCase.java

```diff
@@ -54,7 +54,7 @@ public void convertAndSend(final Order<?> order, String receivedSubmissionId)
                 EtorMetadataStep.ETOR_PROCESSING_TAG_ADDED_TO_MESSAGE_HEADER);
 
         String sentSubmissionId = sender.send(omlOrder).orElse(null);
```

Comment:
Should this `sentSubmissionId` variable also be some flavor of `reportId`?

Reply:
Good catch.

Reply:
Fixed.